python-scripts/xbox/ke.py

```diff
@@ -24,6 +24,9 @@ def AvSendTVEncoderOption(RegisterBase, Option, Param, Result):
 def IoDeviceObjectType():
   return pe.resolve_export(70)
 
+def KeTickCount():
+  return (pe.resolve_export(156))
```

Comment:
Remove outer braces and move this function below IoSynchronousDeviceIoControlRequest
(These functions are supposed to be ordered by kernel ordinal / function index)